src/RequestForm.js

```diff
@@ -1100,6 +1101,7 @@ class RequestForm extends React.Component {
       },
       errorMessage,
       onCancel,
+      isTlrEnabled,
```

Comment:
Could you please check, can we add default value for current variable?

Reply:
done